### ISSUE TITLE
Duplicate search results for aliased articles

### ISSUE THREAD

**Issue by ifarley** [CrossRef/rest-api-doc#399]
When a DOI is aliased, shouldn't the search results at search.crossref.org for those articles only produce one search result (that of the primary DOI)? Here are a few examples where aliased DOIs are appearing as duplicates in search results:

https://t.co/E1wzXfhy1t
https://t.co/d4NCLv0I8x
https://t.co/fSgg8LRwT6
https://t.co/F65OZU8Inz

**Comment by ppolischuk** [comment]
Possible this may not be a REST API issue but instead a search issue. @eshadatta-crossref would you happen to know?

**Comment by gbilder** [comment]
Agree, we should prioritise primary DOI. But as I recall, there is nothing in the XML that the API can use to determine that something is an alias or has been aliased. Important that we have both directions. We don't want to *only* index the primary because then anybody with the old DOI would not be able to find the record. Ideally, we'd include the aliased DOIs in the primary's record so that they could still be found.
